Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2110,15 +2110,15 @@ static Ref<CSSValue> maskModeToCSSValue(MaskMode type)
     return CSSValuePool::singleton().createValue(CSSValueMatchSource);
 }
 
-static Ref<CSSValue> fillSizeToCSSValue(const FillSize& fillSize, const RenderStyle& style)
+static Ref<CSSValue> fillSizeToCSSValue(const FillSize& fillSize, const RenderStyle& style, CSSPropertyID propertyID)
```

Comment:
Can you make the propertyID the first argument to be consistent with ` createPositionListForLayer` and `createSingleAxisPositionValueForLayer`?